docs/reference/mpremote.rst

```diff
@@ -264,24 +281,37 @@ The full list of supported commands are:
     There is no supported way to undelete files removed by ``mpremote rm -r :``.
     Please use with caution.
 
-  All other commands implicitly assume the path is a remote path, but the ``:``
-  can be optionally used for clarity.
+  *mpremote fs tree*
 
-  All of the filesystem sub-commands take multiple path arguments, so if there
-  is another command in the sequence, you must use ``+`` to terminate the
-  arguments, e.g.
+  The ``tree`` command will print a tree of the given directories. 
+  Using the ``--size/-s`` option will print the size of each file, or use 
+  ``--human/-h`` to use a more human readable format. 
+  Note: Directory size is only printed when a non-zero size is reported by the device's filesystem.
+  The ``-v`` option  can be used to include the name of the serial device in 
+  the output.
 
   .. code-block:: bash
 
-      $ mpremote fs cp main.py :main.py + repl
+        $ mpremote tree -hv
+        tree :
+        : /on /dev/ttyACM0
+        ├── [   90B] config.py
+        ├── data
+        │   └── [   122K] norwegian.log
+        ├── lib
+        │   ├── [  2.4K] parrot.py
+        │   └── [  3.4K] worker.py
+        ├── [  800B] main.py
+        └── [  2.5K] readme.md
 
-  This will copy the file to the device then enter the REPL. The ``+`` prevents
-  ``"repl"`` being interpreted as a path.
+  *other fs commands*
 
-  The ``cp`` command supports the ``-r`` option to make a recursive copy.  By
-  default ``cp`` will skip copying files to the remote device if the SHA256 hash
-  of the source and destination file matches.  To force a copy regardless of the
-  hash use the ``-f`` option.
+  All other commands implicitly assume the path is a remote path, but the ``:``
+  can be optionally used for clarity.
+
+  All of the filesystem sub-commands take multiple path arguments, so if there
+  is another command in the sequence, you must use ``+`` to terminate the
+  arguments, e.g.
```

Comment:
This "eg" is missing its example code.  As mentioned above, this would be fixed by not reordering the file, just add a paragraph about tree.